### ISSUE TITLE
When VS Code tries to download the necessary files from the server, the server gives a "404 Not Found" error and cannot start the Docker container

### ISSUE THREAD

**Issue by junpei-9898** [getcursor/cursor#1339]
1) If you can, please include a screenshot of your problem
![スクリーン ショット 2024-03-22 に 13 55 26 午後](https://github.com/getcursor/cursor/assets/83627275/6fbc5a68-b90d-494d-bbdb-5ed5a322cc17)

2) Please include the name of your operating system
macOS 14.3.1

3) If you can, steps to reproduce are super helpful
Fork the repository (https://github.com/geekan/MetaGPT.git)
2. clone from your own branch
3. Start devcontainer
4.The corresponding error occurs.

No particular problem was observed when the above procedure was carried out with vscode. The most recent update of cursor caused the problem, so the most recent update is considered to be the cause of the problem.

**Comment by truell20** [comment]
Thank you for the report. Could you try 0.30.3? Just released a new update that should fix this.

**Comment by truell20** [comment]
Closing for now! Let me know if anything isn't working as expected.

**Comment by diegoacuna** [comment]
@truell20 I'm in 0.35.1 (fresh install) and I have a similar error:

```
2024-07-05T05:48:38.704Z] Start: Run in container: test -d '/home/node/.cursor-server/bin/a915e3254b1192ce240c078da83fc1768a6e3e90'
[2024-07-05T05:48:38.704Z] 
[2024-07-05T05:48:38.704Z] 
[2024-07-05T05:48:38.704Z] Exit code 1
[2024-07-05T05:48:38.704Z] Stop (0 ms): Run in container: test -d '/home/node/.cursor-server/bin/a915e3254b1192ce240c078da83fc1768a6e3e90'
[2024-07-05T05:48:38.705Z] Start: Run in container: test -d '/vscode/cursor-server/bin/linux-arm64/a915e3254b1192ce240c078da83fc1768a6e3e90'
[2024-07-05T05:48:38.705Z] 
[2024-07-05T05:48:38.705Z] 
[2024-07-05T05:48:38.705Z] Exit code 1
[2024-07-05T05:48:38.705Z] Stop (0 ms): Run in container: test -d '/vscode/cursor-server/bin/linux-arm64/a915e3254b1192ce240c078da83fc1768a6e3e90'
[2024-07-05T05:48:38.705Z] Installing VS Code Server for commit a915e3254b1192ce240c078da83fc1768a6e3e90
[2024-07-05T05:48:38.705Z] Start: Run in container: /bin/sh
[2024-07-05T05:48:38.706Z] Start: Downloading VS Code Server
[2024-07-05T05:48:38.706Z] a915e3254b1192ce240c078da83fc1768a6e3e90 linux-arm64 stable
[2024-07-05T05:48:39.231Z] Failed to download VS Code Server (https://update.code.visualstudio.com/commit:a915e3254b1192ce240c078da83fc1768a6e3e90/server-linux-arm64/stable): HTTP 404 - Not Found
[2024-07-05T05:48:39.231Z] Retrying to download VS Code Server.
[2024-07-05T05:48:39.695Z] Stop (989 ms): Downloading VS Code Server
[2024-07-05T05:48:39.701Z] Error: Server returned 404
    at h (vscode-file://vscode-app/Applications/Cursor.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1187:29331)
    at S.download (vscode-file://vscode-app/Applications/Cursor.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1188:1737)
```

**Comment by zachrip** [comment]
This issue is happening again on 0.40.1

**Comment by Abushawish** [comment]
Still happening as of 0.41.1 too

**Comment by meech-ward** [comment]
![Screenshot 2024-10-09 at 2 22 56 PM](https://github.com/user-attachments/assets/75f78642-663e-4bbe-b260-31068d73e272)

**Comment by JQuezada0** [comment]
Same on 0.41.3

```
Version: 0.41.3
VSCode Version: 1.91.1
Commit: 51c8aff7cb5a89f4a0e462fbacab938bdbfaf140
Date: 2024-09-25T17:13:50.377Z
Electron: 29.4.0
ElectronBuildId: undefined
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Darwin arm64 23.6.0
```

**Comment by hhh2210** [comment]
> This issue is happening again on 0.40.1

not fixed at 0.44.8
